modules/swagger-codegen/src/main/resources/Java/build.gradle.mustache

```diff
@@ -19,6 +19,7 @@ repositories {
 }
 
 
+
```

Comment:
Not sure why we need 3 empty lines here.

Reply:
you are right, i removed 2 lines now because of the other formattings in the file